aries-site/src/examples/templates/table-customization/components/ColumnSettings.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { Box, CheckBoxGroup, List, Tab, Tabs, TextInput } from 'grommet';
+import { Search } from 'grommet-icons';
+
+export const ColumnSettings = ({
+  columns,
+  setColumns,
+  visible,
+  setVisible,
+  ...rest
+}) => {
+
+  const [filteredColumns, setFilteredColumns] = useState(columns);
+  const [search, setSearch] = useState('');
+
+  const filterColumns = (array, searchValue = search) => {
+    
+    let filterResults = array;
+
+    if (searchValue) {
+      filterResults = array.filter(o =>
+        Object.keys(o).some(
+          k =>
+            typeof o[k] === 'string' &&
+            o[k].toLowerCase().includes(searchValue.toLowerCase()),
+        ),
+      );
+    }
+    return filterResults;
+  };
+
+  return (
+    <Box pad="small" elevation="medium">
+      <Tabs {...rest}>
+        <Tab title="Select Columns">
```

Comment:
Occasionally, I am seeing the Tabs wrapped.

![Screen Shot 2021-05-27 at 2 33 57 PM](https://user-images.githubusercontent.com/1756948/119893552-be5f1000-bef8-11eb-9f92-e82390eeac4d.png)